src/pages/home/slider-home-page/slider-home-page.js

```diff
@@ -1,38 +1,31 @@
 import React from 'react';
 import AwesomeSlider from 'react-awesome-slider';
 import 'react-awesome-slider/dist/styles.css';
-import { useSelector } from 'react-redux';
 import { useStyles } from './slider-home-page.style';
-import { getImage } from '../../../utils/imageLoad';
-import { Backdrop } from '@material-ui/core';
-import LoadingBar from '../../../components/loading-bar';
+import { useSelector } from 'react-redux';
+import withAutoplay from 'react-awesome-slider/dist/autoplay';
+import { carouselInterval } from '../../../configs';
+
+const AutoplaySlider = withAutoplay(AwesomeSlider);
 
 const SliderHomePage = () => {
   const styles = useStyles();
-  const { models, loading } = useSelector(({ Model }) => ({
-    models: Model.models,
-    loading: Model.loading
+  const { categories } = useSelector(({ Categories: { list } }) => ({
+    categories: list.filter((element) => element.isMain)
   }));
-  if (loading) {
-    return (
-      <Backdrop className={styles.backdrop} open={loading} invisible>
-        <LoadingBar color='inherit' />
-      </Backdrop>
-    );
-  }
-
-  return models.length > 0 ? (
+  return (
     <div className={styles.captionBlock}>
-      <AwesomeSlider className={styles.slider} mobileTouch>
-        {models.map((photo) => (
-          <div
-            key={photo.name[1].value}
-            data-src={getImage(photo.images.large)}
-          />
+      <AutoplaySlider
+        play={true}
+        interval={carouselInterval}
+        className={styles.slider}
+      >
+        {categories.map(({ images }) => (
+          <div key={images.large} data-src={images.large} />
```

Comment:
You still setting wrong image src. Please, import IMG_URL and concatenate with yours data.

Reply:
Fixed